easybuild/easyconfigs/p/python-pulp/python-pulp-2.5.1-foss-2021a.eb

```diff
@@ -0,0 +1,35 @@
+# Contribution by
+# DeepThought, Flinders University
+# R.QIAO <rob.qiao@flinders.edu.au>
+
+easyblock = 'PythonPackage'
+
+name = 'python-pulp'
+version = '2.5.1'
+
+homepage = 'https://github.com/coin-or/pulp'
+description = """
+PuLP is an LP modeler written in Python. PuLP can generate MPS or LP files and
+call GLPK, COIN-OR CLP/CBC, CPLEX, GUROBI, MOSEK, XPRESS, CHOCO, MIPCL, SCIP to
+solve linear problems.
+"""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+toolchainopts = {'usempi': True}
+
+source_urls = ['https://pypi.python.org/packages/source/%(nameletter)s/pulp']
+sources = ['PuLP-%(version)s.tar.gz']
+checksums = ['27c2a87a98ea0e9a08c7c46e6df47d6d4e753ad9991fea2901892425d89c99a6']
+
+dependencies = [
+    ('Python', '3.9.5'),
+]
```

Comment:
We can add dependencies to Cbc and GLPK, which are easily available and already have easyconfigs. I would also add a dependency on Gurobi, but leave it commented out as it requires a specific license.

Reply:
@lexming good spot, thx.

Reply:
> We can add dependencies to Cbc and GLPK, which are easily available and already have easyconfigs. I would also add a dependency on Gurobi, but leave it commented out as it requires a specific license.

@lexming done.